src/vma/proto/dst_entry.h

```diff
@@ -77,15 +77,7 @@ class dst_entry : public cache_observer, public tostr, public neigh_observer
 	in_addr_t	get_dst_addr();
 	uint16_t	get_dst_port();
 	inline in_addr_t get_src_addr() const {
```

Comment:
We agreed in meeting to keep it inline and that what is done. In general this function was not inlined before pull request changes so we can return it back.